package.json

```diff
@@ -13,12 +13,12 @@
     "author": "Globality Engineering",
     "license": "MIT",
     "dependencies": {
-        "aws-sdk": "^2.231.1",
+        "aws-sdk": "^2.829.0",
```

Comment:
**Comment:**

The current policy for dependencies is to lock to specific versions whilst we are working out a long term dependency management solution.

Can you please remove the carat from them. eg `aws-sdk: "2.829.0"`